app/src/main/res/layout/fragment_login.xml

```diff
@@ -4,117 +4,134 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
     android:gravity="center"
-    android:theme="@style/Theme.StudentApp"
-    android:padding="60dp"
+    android:orientation="vertical"
+    android:padding="16dp"
     tools:context=".auth.LoginFragment">
 
     <androidx.constraintlayout.helper.widget.Flow
```

Comment:
We have abandoned the Flow container, but it is not critical, let it remain. If anything, I'll fix it after the merger